src/styles/main.scss

```diff
@@ -1,3 +1,56 @@
+* {
+  box-sizing: border-box;
+}
+
+
+$days: ('mon':0, 'tue':1, 'wed':2, 'thu':3, 'fri':4, 'sat':5, 'sun':6);
+$cell_width:100px;
+$gap:1px;
+$month_length:(28,29,30,31);
```

Comment:
```suggestion
$cell_width: 100px;
$gap: 1px;
$month_length: (28,29,30,31);
```

Reply:
Why you didn't fix this comments, if you think that this comment wrong you have to discuss it with mentor

![image](https://user-images.githubusercontent.com/63306833/235625796-4c6f1473-f908-4f02-8c6b-beeb0883f024.png)